openlane/steps/odb.py

```diff
@@ -501,8 +503,9 @@ class HeuristicDiodeInsertion(OdbpyStep):
         Variable(
             "HEURISTIC_ANTENNA_THRESHOLD",
             Optional[Decimal],
-            "A manhattan distance above which a diode is recommended to be inserted by a heuristic inserter. If not specified, the heuristic inserter will typically use a default value.",
+            "A manhattan distance above which a diode is recommended to be inserted by a heuristic inserter. If not specified, the heuristic inserter will use a default value equal to 200x the minimum site width in the technology files.",
             units="µm",
+            pdk=True,
```

Comment:
I think we need to add this to the PDK and update it for consistency and make it non optional value